### ISSUE TITLE
ble.enum fails with "ble.recon connection timeout" on bettercap 2.32.0

### ISSUE THREAD

**Issue by kadin2048** [bettercap/bettercap#972]
Bettercap 2.32.0 seems to consistently fail enumerating BLE devices on my machine, which is running a clean install of Debian Bullseye 

### Environment
* Bettercap version you are using ( `bettercap -version` ).
`bettercap v2.32.0 (built for linux amd64 with go1.15.15)`
* OS version and architecture you are using.
Debian 11 "Bullseye" with all updates as of 2022-06-24
Kernel info: `Linux thinkpad 5.10.0-15-amd64 #1 SMP Debian 5.10.120-1 (2022-06-09) x86_64 GNU/Linux`
I also tested using the Docker version (bettercap/bettercap which is also 2.32.0) and got the same result
Hardware is a Lenovo T430s with onboard OEM Bluetooth, reports as `0a5c:21e6 Broadcom Corp. BCM20702 Bluetooth 4.0 [ThinkPad]`
* Go version if building from sources.
`go1.15.15` (packaged with Debian, `apt install golang`)
* Command line arguments you are using.
Running `sudo bettercap -debug` and also tested using web UI, got the same results
* Caplet code you are using or the interactive session commands.
See debug log or Steps to Reproduce, below, for interactive commands.
* **Full debug output** while reproducing the issue ( `bettercap -debug ...` ).
Debug output available in this Gist: https://gist.github.com/kadin2048/a72b2d5c0c41dbfd2c8c046438e21863

### Steps to Reproduce

1. `ble.recon on`
2. wait a while for devices to populate
3. `ble.recon off`
4. `ble.show`
5. `ble.enum [MAC address]`
6. usually a 10s pause or so, on the first run
7. error message `ble.recon connection timeout` is displayed alongside debug output

**Expected behavior:**
Would have expected enum output for the device specified by the MAC address.

**Actual behavior:**
Receive message `[sys.log] [war] ble.recon connection timeout`
Issue is repeatable, and appears to happen faster on subsequent tries after the initial one

Happy to provide additional information that might help with resolving the issue.

**Comment by ShadowRL76** [comment]
Hello, Wondering if you ever found a solution. As I am running bettercap and also getting the same error.

**Comment by Jachimo** [comment]
No, unfortunately I haven't made any progress on it.  (I ended up using a different tool for the short-term need, but would still like to get Bettercap working.)

Out of curiosity, what hardware/software environment are you using?  Is it the BCM20702 chipset by chance? 

Would be nice if we could at least pin this down to the hardware if that's where it's coming from.

**Comment by kadin2048** [comment]
I just retested and it's still doing the same thing. `ble.enum` followed by the address of any device just causes a connection timeout error. Output is identical to the log from June, linked above.

Are you using the same Bluetooth chipset in your machine? The BCM20702?

**Comment by zeyadmbk** [comment]
Hi all, wondering the same thing if anyone got this working.

**Comment by Centaro0** [comment]
+1

bettercap v2.41.0 (built for linux amd64 with go1.24.0)
